src/index.js

```diff
@@ -1,46 +1,157 @@
-// import _ from 'lodash';
 import './style.css';
-// Images for projects
+import Store from './modules/Store.js';
+import Task from './modules/task.js';
 
+// Images for projects
+import Bin from './images/bin.png';
 import Dots from './images/dots.png';
 import Enter from './images/enter.png';
 import Refresh from './images/refresh.png';
 // Add images in html.
 
-const refreshContainer = document.querySelector('.refresh-list');
+const refreshContainer = document.querySelector('.refresh-container');
 const refresh = new Image();
 refresh.src = Refresh;
 refreshContainer.innerHTML = `<img src=${Refresh} class="refresh-icon" />`;
 
-const enterContainer = document.querySelector('.enter-list');
+const enterContainer = document.querySelector('.enter-container');
 const enter = new Image();
 enter.src = Enter;
 enterContainer.innerHTML = `<img src=${Enter} class="enter-icon" />`;
 
-const listContainer = document.querySelector('.list-container');
 const dots = new Image();
 dots.src = Dots;
-// Array with String Bool number.
-const tasks = [
-  { index: 0, description: 'Read for 30 Minutes', completed: true },
-  { index: 1, description: 'Exersisce for 30 Minutes', completed: false },
-  { index: 2, description: 'Complete Today Projects', completed: false },
-];
-// create New Task Array for new list
-const createTask = (tasks) => {
-  const taskContent = `
-  <li class="element">
-      <input type="checkbox" id="checkbox" class="checkbox" />
-      <div>${tasks.description}</div>
-      <div class="dots-container"><img class="dots"src=${Dots} /></div>
-  </li>`;
-  // added task in html list
-  listContainer.innerHTML += taskContent;
-};
-// create new index for each new task list
-const createList = (arr) => {
-  for (let i = 0; i < arr.length; i += 1) {
-    createTask(arr[i]);
-  }
-};
-createList(tasks);
\ No newline at end of file
+
+const store = new Store();
+
+function getUItasks() {
+  const UiTasks = document.querySelectorAll('.element');
+  return UiTasks;
+}
+
+function updateTask(e) {
+  const text = e.target.value;
+  const value = e.target.id;
+  const splitid = value.split('-');
+  const idstring = splitid[2];
+  const id = parseInt(idstring, 10);
+  const list = store.getList();
+  for (let i = 0; i < list.length; i += 1) {
+    const task = list[i];
+    if (task.id === id) {
+      list[i].description = text;
+    }
+  }
+  localStorage.setItem('list', JSON.stringify(list));
+}
```

Comment:
- When I try updating a task, it doesn't get updated in the local storage, please fix this issue 🙏 🙏 

Your project:
<img width="959" alt="wdw" src="https://user-images.githubusercontent.com/73607512/202215919-b6dc85d3-43b4-4940-a567-96b7ee0ca7dc.PNG">